src/theme/GlobalStyles.js

```diff
@@ -1,10 +1,23 @@
 import { createGlobalStyle } from 'styled-components';
 
 const GlobalStyles = createGlobalStyle`
+    :root {
+        font-family: 'Poppins', sans-serif;
+        font-size: 62.5%; 
```

Comment:
I think that it's better to use a `em` or `rem` unit.

Reply:
Yes we will use it. It's only reset on html element that let us easier calculate `rem`. Setting it up we get `1rem` = `10px`